### PR TITLE
cmd: Minor changes to make ftests easier to write

### PR DESCRIPTION
**Pull request by mattyw**

### CODE REVIEW

File: cmd/listbudgets/list-budgets.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ import (
 
 // NewListBudgetsCommand returns a new command that is used
 // to list budgets a user has access to.
-func NewListBudgetsCommand() cmd.Command {
+func NewListBudgetsCommand() modelcmd.CommandBase {
```
Comment (alesstimec) [inline cmd/listbudgets/list-budgets.go]: why not return *listBudgetsCommand?

File: cmd/showbudget/show_budget.go [hunk 2]
```diff
@@ -19,7 +19,7 @@ import (
 
 // NewShowBudgetCommand returns a new command that is used
 // to show details of the specified wireformat.
-func NewShowBudgetCommand() cmd.Command {
+func NewShowBudgetCommand() modelcmd.CommandBase {
```
Comment (alesstimec) [inline cmd/showbudget/show_budget.go]: same question

**Comment by alesstimec** [comment]
LGTM, but why don't we return pointers to instantiated commands?

File: cmd/updateallocation/updateallocation.go [hunk 3]
```diff
@@ -24,7 +24,7 @@ type updateAllocationCommand struct {
 }
 
 // NewUpdateAllocationCommand returns a new updateAllocationCommand.
-func NewUpdateAllocationCommand() cmd.Command {
+func NewUpdateAllocationCommand() modelcmd.ModelCommand {
```
Comment (tasdomas) [inline cmd/updateallocation/updateallocation.go]: Why not modelcmd.CommandBase as in the other changes?

Comment (mattyw) [inline cmd/updateallocation/updateallocation.go]: @tasdomas because they're different types

**Comment by cmars** [comment]
LGTM

**Comment by mattyw** [comment]
$$merge$$

**Pull request merged**